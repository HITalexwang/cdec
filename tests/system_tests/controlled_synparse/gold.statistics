-lm_nodes 11
-lm_edges 18
-lm_paths 18
-lm_trans a d c b
-lm_trans 0
constr_nodes 8
constr_edges 8
constr_paths 1
-lm_nodes 11
-lm_edges 18
-lm_paths 18
-lm_trans a d c b
-lm_trans 0
constr_nodes 12
constr_edges 14
constr_paths 3
