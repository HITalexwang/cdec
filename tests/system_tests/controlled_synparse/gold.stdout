0 ||| c b c d ||| F1bad2=1;F5=1;F6=1;F7=1;Glue=3
0 ||| c d c b e ||| F1bad2=1;F4=1;F5=1;F7=1;Glue=1
0 ||| c b c d ||| F1bad2=1;F6=1;F7=1;F9=1;Glue=2
0 ||| b d c b e ||| F1bad1=1;F4=1;F5=1;F7=1;Glue=1
0 ||| c b c d ||| F1bad2=1;F3=1;F5=1;F7=1;Glue=1
0 ||| a d c b e ||| F1=1;F4=1;F5=1;F7=1;Glue=1
0 ||| b b c d ||| F1bad1=1;F3=1;F5=1;F7=1;Glue=1
0 ||| c d c b ||| F1bad2=1;F2=1;F5=1;F7=1;Glue=1
0 ||| a b c d ||| F1=1;F3=1;F5=1;F7=1;Glue=1
0 ||| b b c d ||| F1bad1=1;F5=1;F6=1;F7=1;Glue=3
0 ||| b d c b ||| F1bad1=1;F2=1;F5=1;F7=1;Glue=1
0 ||| a d c b ||| F1=1;F2=1;F5=1;F7=1;Glue=1
0 ||| b b c d ||| F1bad1=1;F6=1;F7=1;F9=1;Glue=2
0 ||| c c b d ||| F1bad2=1;F6=1;F7=1;F8=1;Glue=2
0 ||| a b c d ||| F1=1;F6=1;F7=1;F9=1;Glue=2
0 ||| a b c d ||| F1=1;F5=1;F6=1;F7=1;Glue=3
0 ||| b c b d ||| F1bad1=1;F6=1;F7=1;F8=1;Glue=2
0 ||| a c b d ||| F1=1;F6=1;F7=1;F8=1;Glue=2
1 ||| c b c d ||| F1bad2=1;F5=1;F6=1;F7=1;Glue=3
1 ||| c d c b e ||| F1bad2=1;F4=1;F5=1;F7=1;Glue=1
1 ||| c b c d ||| F1bad2=1;F6=1;F7=1;F9=1;Glue=2
1 ||| b d c b e ||| F1bad1=1;F4=1;F5=1;F7=1;Glue=1
1 ||| c b c d ||| F1bad2=1;F3=1;F5=1;F7=1;Glue=1
1 ||| a d c b e ||| F1=1;F4=1;F5=1;F7=1;Glue=1
1 ||| b b c d ||| F1bad1=1;F3=1;F5=1;F7=1;Glue=1
1 ||| c d c b ||| F1bad2=1;F2=1;F5=1;F7=1;Glue=1
1 ||| a b c d ||| F1=1;F3=1;F5=1;F7=1;Glue=1
1 ||| b b c d ||| F1bad1=1;F5=1;F6=1;F7=1;Glue=3
1 ||| b d c b ||| F1bad1=1;F2=1;F5=1;F7=1;Glue=1
1 ||| a d c b ||| F1=1;F2=1;F5=1;F7=1;Glue=1
1 ||| b b c d ||| F1bad1=1;F6=1;F7=1;F9=1;Glue=2
1 ||| c c b d ||| F1bad2=1;F6=1;F7=1;F8=1;Glue=2
1 ||| a b c d ||| F1=1;F6=1;F7=1;F9=1;Glue=2
1 ||| a b c d ||| F1=1;F5=1;F6=1;F7=1;Glue=3
1 ||| b c b d ||| F1bad1=1;F6=1;F7=1;F8=1;Glue=2
1 ||| a c b d ||| F1=1;F6=1;F7=1;F8=1;Glue=2
